buildbot.mariadb.org/scripts/rpm-install.sh

```diff
@@ -1,79 +1,80 @@
-#!/bin/bash
+#!/usr/bin/env bash
+# shellcheck disable=SC2154
+
 set -ex
+# //TEMP why vlad?
 branch=${1:-bb-10.8-release}
 df -kT
+
+# //TEMP this should be done in the VM preparation IMO
 case "$branch" in
-*mdev10416*)
-  sudo cat /etc/sysconfig/selinux | grep SELINUX || true
-  sudo sh -c \"PATH=$PATH:/usr/sbin getenforce || true\"
-  sudo sh -c \"PATH=$PATH:/usr/sbin setenforce Enforcing || true\"
-  sudo sh -c \"PATH=$PATH:/usr/sbin getenforce || true\"
-  ;;
+  *mdev10416*)
```

Comment:
@vladbogo we need to handle this properly. I suggest that we have specific tests to cover this if still needed... Or that we configure our VM with enforcing by default.

Reply:
not sure that I understand. Isn't it something needed just for some mdev until solved? We should sync with the scripts from the old bb though since I assume there might have been some updates there

Reply:
It looks like a test of selinux being enforcing. Selinux should always be enforcing during our tests to represent a real user environment.

The only unknown here is if columnstore is fully ready for it.

Reply:
Yep, this is my understanding too. I will make sure that it is enforced by default on the new VMs.